### PR TITLE
Release changes

### PR DESCRIPTION
**Pull request by JainiDarji**
Comment addressed from review

### CODE REVIEW

File: RedpenCapture/Utility/Constants.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ internal struct Constants {
     static let redpenWidget = "RedpenWidget"
     static let imageExtension = ".png"
     static let dateFormat = "dd/MM/yyyy HH:mm:ss"
-    static let methodChannelName = "redpen.flutter.widget/image"
+    static let methodChannel = "redpen.flutter.widget/image"
     static let widgetId = "widgetId"
     static let takeScreenShort = "takeScreenShort"
```
Comment (milaniterate) [inline RedpenCapture/Utility/Constants.swift:15]: takeScreenShot

File: RedpenCapture/MotionDetection.swift [hunk 2]
```diff
@@ -26,7 +26,7 @@ internal class MotionDetection: MotionSensorDelegate {
     internal init() {
         self.motionManager = CMMotionManager()
         if motionManager.isAccelerometerAvailable {
-            motionManager.accelerometerUpdateInterval = Constants.updateInterval
+            motionManager.accelerometerUpdateInterval = Redpen.sharedInstance.getRedpenWidgetInstance()?.redpenWidgetConfig.accelerometerUpdateInterval ?? Constants.updateInterval
```
Comment (milaniterate) [inline RedpenCapture/MotionDetection.swift:29]: We dont need to check optional redpenWidgetConfig.accelerometerUpdateInterval will always have value

File: RedpenCapture/MotionDetection.swift [hunk 3]
```diff
@@ -46,12 +46,20 @@ internal class MotionDetection: MotionSensorDelegate {
...
+            registerDetection()
+        }
     }
     
     /// Registers accelerometer updates and triggers the delegate on device shake.
     internal func registerDetection() {
+        guard motionManager.isAccelerometerAvailable else { return }
         motionManager.startAccelerometerUpdates(to: .main, withHandler: { [weak self] (data, error) in
+            if let error = error {
+                print("Accelerometer update error: \(error.localizedDescription)")
```
Comment (milaniterate) [inline RedpenCapture/MotionDetection.swift:59]: Remove unwanted print statemnt, Use Logger if needed

File: RedpenCapture/MotionDetection.swift [hunk 4]
```diff
@@ -46,12 +46,20 @@ internal class MotionDetection: MotionSensorDelegate {
...
+                return
+            }
+            
             guard let acceleration = data?.acceleration else {
                 return
             }
```
Comment (milaniterate) [inline RedpenCapture/MotionDetection.swift:65]: At line number 68 can we guard self and use that one

File: RedpenCapture/MotionDetection.swift [hunk 5]
```diff
@@ -67,6 +75,6 @@ internal class MotionDetection: MotionSensorDelegate {
     ///
     /// - Returns: The accelerometer update interval.
     private func getUpdateInterval() -> Double {
```
Comment (milaniterate) [inline RedpenCapture/MotionDetection.swift:77]: After correct the updateInterval we will not require this method as well

File: RedpenCapture/MotionDetection.swift [hunk 6]
```diff
@@ -46,12 +46,20 @@ internal class MotionDetection: MotionSensorDelegate {
     /// Resets motion detection by stopping and registering again.
     func resetDetection() {
         stopDetection()
-        registerDetection()
+        if sensorEventDelegate != nil {
```
Comment (milaniterate) [inline RedpenCapture/MotionDetection.swift:49]: check with variable codition

File: RedpenCapture/RedpenWidgetInstance.swift [hunk 7]
```diff
@@ -38,7 +38,9 @@ internal class RedpenWidgetInstance {
```
Comment (milaniterate) [inline RedpenCapture/RedpenWidgetInstance.swift:38]: Line number 31 can we move that line to other place, our function is lieing

File: RedpenCapture/RedpenWidgetInstance.swift [hunk 8]
```diff
@@ -38,7 +38,9 @@ internal class RedpenWidgetInstance {
     
     /// Presents the user action view and handles the callback.
     private func presentUserActionView() {
-        presenterView.presentView(completionHandler: { success in
+        presenterView.presentView(completionHandler: { [weak self] success in
+            guard let self = self else { return }
+            
             if(!success) {
```
Comment (milaniterate) [inline RedpenCapture/RedpenWidgetInstance.swift:44]: can we use single line condition

File: RedpenCapture/Utility/Constants.swift (hunk 1 above)
Comment (JainiDarji) [reply RedpenCapture/Utility/Constants.swift:15]: Modified

File: RedpenCapture/MotionDetection.swift (hunk 2 above)
Comment (JainiDarji) [reply RedpenCapture/MotionDetection.swift:29]: Modified

File: RedpenCapture/MotionDetection.swift (hunk 3 above)
Comment (JainiDarji) [reply RedpenCapture/MotionDetection.swift:59]: Updated

File: RedpenCapture/RedpenWidgetInstance.swift (hunk 8 above)
Comment (JainiDarji) [reply RedpenCapture/RedpenWidgetInstance.swift:44]: Updated

File: RedpenCapture/MotionDetection.swift (hunk 6 above)
Comment (JainiDarji) [reply RedpenCapture/MotionDetection.swift:49]: Updated

File: RedpenCapture/RedpenWidgetInstance.swift (hunk 7 above)
Comment (JainiDarji) [reply RedpenCapture/RedpenWidgetInstance.swift:38]: Moved to other place

File: RedpenCapture/MotionDetection.swift (hunk 4 above)
Comment (JainiDarji) [reply RedpenCapture/MotionDetection.swift:65]: Modified

File: RedpenCapture/MotionDetection.swift (hunk 5 above)
Comment (JainiDarji) [reply RedpenCapture/MotionDetection.swift:77]: Updated

**Pull request merged**